pan3d/ui/analytics.py

```diff
@@ -0,0 +1,405 @@
+from trame.decorators import TrameApp, change
+from trame.widgets import vuetify3 as v3, plotly, html
+
+import hashlib
+
+try:
+    import xcdat  # noqa
+except ModuleNotFoundError as e:
+    print(
+        f"""
+        Error occurred while importing xcdat required for Analytics Explorer: {e}
+        Please make sure xcdat is installed properly using conda -- `conda install -c conda-forge xcdat`
+        You may want to create a new conda environment an reinstall pan3d within along with xcdat.
+        """
```

Comment:
is `xcdat` available on PyPI?

Reply:
It is not, it's only available through conda-forge at this point.

Reply:
crap, to bad...